app/scripts/lockdown-run.js

```diff
@@ -6,6 +6,7 @@ try {
     errorTaming: 'unsafe',
     mathTaming: 'unsafe',
     dateTaming: 'unsafe',
+    domainTaming: 'unsafe',
```

Comment:
Added due to this breaking change:

https://github.com/endojs/endo/blob/master/packages/ses/CHANGELOG.md#0150-2021-11-02